server/middleware/legalSenderJourneyAuthenticationStartPage.ts

```diff
@@ -0,0 +1,5 @@
+import config from '../config'
+
+export default function legalSenderJourneyAuthenticationStartPage() {
```

Comment:
Function to correctly return the correct start page URL for the Legal Sender Journey based on the value of the feature flag (ie. either use magic link or One Time Code urls/routes) - means we dont need to replicate the logic in several places